data/src/test/java/org/apache/iceberg/data/TestDataFileIndexStatsFilters.java

```diff
@@ -94,24 +94,24 @@ public void createTableAndData() throws IOException {
             .filter(rec -> rec.getField("category").equals("even"))
             .collect(Collectors.toList());
 
-    this.dataFile = FileHelpers.writeDataFile(table, Files.localOutput(temp.newFile()), records);
+    this.dataFile = FileHelpers.writeDataFile(table, Files.localOutput(temp.toFile()), records);
```

Comment:
as I mentioned in my earlier comment, this should be `this.dataFile = FileHelpers.writeDataFile(table, Files.localOutput(File.createTempFile("junit", null, temp.toFile())), records);`

Reply:
@igoradulian it appears that this hasn't been addressed yet